spec/services/idv/inherited_proofing/va/service_spec.rb

```diff
@@ -62,5 +62,63 @@
         it_behaves_like 'an invalid auth code error is raised'
       end
     end
+
+    context 'when a request error is raised' do
+      before do
+        allow(service).to receive(:request).and_raise('Boom!')
+      end
+
+      it 'rescues and returns the error' do
+        expect(service.execute).to eq({ service_error: 'Boom!' })
+      end
+    end
+
+    context 'when a decryption error is raised' do
+      it 'rescues and returns the error' do
+        freeze_time do
+          stub_request(:get, request_uri).
+            with(headers: request_headers).
+            to_return(status: 200, body: 'xyz', headers: {})
+
+          expect(service.execute.to_s).to match(/unexpected token at 'xyz'/)
```

Comment:
Doesn't this return a hash? We should actually inspect the hash itself an make sure it contains what we expect and not stringify and match against it with a regex

Reply:
Correct. Not quite sure _what_ I was thinking there TBH lol

Reply:
Addressed.